websites/S/SteamPrices.com/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "EzzChats",
+    "id": "256009740239241216"
+  },
+  "url": "www.steamprices.com",
+  "description": {
+    "en": "SteamPrices.com is a website that enables anyone to compare prices between regions, such as the US, UK, and many more."
+  },
+  "service": "SteamPrices.com",
+  "version": "1.0.0",
+  "logo": "https://img.ezz.moe/0521/12-46-13.png",
+  "thumbnail": "https://img.ezz.moe/0520/20-43-29.png",
```

Comment:
upload to imgur

Reply:
Moved to imgur in https://github.com/PreMiD/Presences/pull/1578/commits/45180b31a9af110db971c412ad7525881c888294